src/codeu/chat/client/simplegui/ChatGuiFX.java

```diff
@@ -56,65 +59,103 @@ public void start(Stage primaryStage) throws Exception {
+        passHBox.setAlignment(Pos.CENTER);
 
-        signInLabel = new Label("Sign-in screen");      // Initialize label
-        signInLabel.setFont(Font.font(20));                 // Set style
+        Label signInLabel = new Label("Sign-in screen");       // Title for sign-in page
+        Label userLabel = new Label("Username:");
+        Label passLabel = new Label("Password:");
+        signInLabel.setFont(Font.font(20));                         // Set style
+        userLabel.setFont(Font.font(15));
+        passLabel.setFont(Font.font(15));
 
         signInButton = new Button("Sign in");                      // Initialize sign in button
-        signInButton.setOnAction((event)-> ButtonClicked(event));       // Initialize its event handler
+        signInButton.setOnAction((event)-> buttonClicked(event));       // Initialize its event handler
 
 
-        userInput = new TextField("Username");
-        passInput = new TextField("Password");
+        userInput = new TextField();
+        passInput = new PasswordField();                        // Set up password fields
+        userInput.setPromptText("Username");                    // TODO: figure out if have the prompt text is overkill
+        passInput.setPromptText("Password");
+        userInput.setAlignment(Pos.CENTER);
+        passInput.setAlignment(Pos.CENTER);
+
 
         signInLabelPane.getChildren().add(signInLabel);         // Add labels to their respective panes
 
+        usernameHBox.getChildren().add(userLabel);
+        usernameHBox.getChildren().add(userInput);          // Set up HBoxes to hold username and password labels/inputs
+        passHBox.getChildren().add(passLabel);
+        passHBox.getChildren().add(passInput);
+
+        inputVBox.getChildren().add(usernameHBox);
+        inputVBox.getChildren().add(passHBox);              // Add those HBoxes to a VBox to stack them on top of each other
+
+        inputMasterBox.getChildren().add(inputVBox);
+        inputMasterBox.getChildren().add(signInButton);     // Add that VBox and the signInButton to the inputMasterBox
+
         signInPane.setTop(signInLabelPane);
-        signInPane.setCenter(signInButton);                 // Add buttons and labels to panes
+        signInPane.setCenter(inputMasterBox);                 // Add label and input box to the pane
 
 
-        signInScene = new Scene(signInPane, WIDTH, HEIGHT);
+        signInScene = new Scene(signInPane, WINDOW_WIDTH, WINDOW_HEIGHT);
 
 
 
         // Main Page
 
-        HBox hboxClient = new HBox();
-        HBox hboxInput = new HBox();
-        VBox userVBox = new VBox();
-        VBox chatVBox = new VBox();
-        VBox convosVBox = new VBox();
-        BorderPane container = new BorderPane();
-        Button send = new Button("Send");
-        Button update = new Button("Update");
-        Button addConvo = new Button("Add Conversation");
+        HBox hboxClient = new HBox();                           // holds the client
```

Comment:
Some of these comments aren't really necessary, and there are places that **should** have comments that don't. Generally I'd try to comment on all of the class-level variables (explain what they're used for, not what they are), but you don't have to comment on every single local variable.